.circleci/config.yml

```diff
@@ -141,6 +141,26 @@ jobs:
             git commit -m "Update document via CircleCI"
             git push -q https://${GITHUB_TOKEN}@github.com/aws/aws-sdk-ios.git  
 
+  run_ui_test:
+    # Specify the Xcode version to use
+    macos:
+      xcode: "10.1.0"
+    steps:
+      - checkout
+      - run:
+          name: Run Sample UI Test
+          command: |
+            git clone https://github.com/awslabs/aws-sdk-ios-samples.git
+            cfolder=$(pwd)
```

Comment:
can we name it something like `rootDirectory`?

Reply:
Renamed it appropriately.